changelog.txt

```diff
@@ -2,6 +2,7 @@ V.Next
 ----------
 - Add refresh_on to access tokens(#1190)
 - Add requested_claims to access tokens, and allow credentials to be filtered by RC (#1187)
+- Adds mock response for cloud discovery to testutils
```

Comment:
you probably don't need to update changelog as this change was done in a test utility and won't be released

Reply:
Oh, I see. Thanks.